### PR TITLE
apg: remove

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@Cogitri please test if no gnome-control-center functionality is lost

**Comment by Cogitri** [comment]
Will do

**Comment by Cogitri** [comment]
Can confirm apg isn't required anymore, it has been replaced with cracklib (which should be added as dep instead).

**Comment by maxice8** [comment]
On Tue, Feb 26, 2019 at 10:15:54AM -0800, Rasmus Thomsen wrote:
> Can confirm apg isn't required anymore, it has been replaced with cracklib (which should be added as dep instead).
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9176#issuecomment-467551346

replaced

File: srcpkgs/gnome-control-center/template [hunk 1]
```diff
@@ -15,8 +15,9 @@ makedepends="ModemManager-devel accountsservice-devel alsa-plugins
  pulseaudio-devel samba-devel tzdata upower-devel vala-devel clutter-gtk-devel
  libxml2-devel libX11-devel gtk+3-devel cups-devel libgudev-devel libwacom-devel
  clutter-devel libsecret-devel libcanberra-devel $(vopt_if cheese cheese-devel)"
-depends="alsa-plugins-pulseaudio apg desktop-file-utils
- gsettings-desktop-schemas hicolor-icon-theme iso-codes sound-theme-freedesktop"
+depends="alsa-plugins-pulseaudio desktop-file-utils
```
Comment (Cogitri) [inline srcpkgs/gnome-control-center/template]: I suppose cracklib could just the added to this line, but oh well

Comment (Cogitri) [reply srcpkgs/gnome-control-center/template]: Maybe also mention that apg has been replaced with cracklib in the commit msg

**Pull request merged**